apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.SessionPoolKey.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.Spanner.Common.V1;
+using System;
+
+namespace Google.Cloud.Spanner.V1;
+public sealed partial class SessionPool
+{
+    /// <summary>
+    /// Aspects under which the sessions are pooled.
+    /// </summary>
+    public class SessionPoolKey : IEquatable<SessionPoolKey>
+    {
+        /// <summary>
+        /// The database for which sessions are pooled on the associated session pool.
+        /// </summary>
+        public DatabaseName DatabaseName { get; }
+
+        /// <summary>
+        /// The database role of the sessions pooled on the associated session pool.
+        /// </summary>
+        public string DatabaseRole { get; }
+
+        /// <summary>
+        ///  Instantiates a new instance of the <see cref="SessionPoolKey"/> object.
```

Comment:
```suggestion
        ///  Creates a new instance of the <see cref="SessionPoolKey"/> object.
```

Reply:
Either change "object" to "class", or just `Creates a new <see cref="SessionPoolKey" />.`